src/web/twig/Extension.php

```diff
@@ -944,11 +944,20 @@ public function shuffleFunction($arr)
      * @param bool|null $namespace Whether class names and IDs within the SVG
      * should be namespaced to avoid conflicts with other elements in the DOM.
      * By default the SVG will only be namespaced if an asset or markup is passed in.
-     * @param string|null $class A CSS class name that should be added to the `<svg>` element.
+     * @param array|string|null $attributes A list of attributes that should be added to the root `<svg>` element.
```

Comment:
`$attributes` should be `$attr`, which should only allow `array|null`, not `string`, and be placed *after* `$class` for BC.

Reply:
Yea, the reason I added `string` was because `$attributes` took the position of `$class` - so BC meant it __could__ have taken in strings. Your solution is better.